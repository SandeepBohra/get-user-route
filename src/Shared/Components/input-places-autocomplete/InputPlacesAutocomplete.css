.InputPlacesAutocomplete input{
    width: 80%;
    padding: 8px;
    border: 1px solid black;
}

.InputPlacesAutocomplete p {
    font-size: 16px;
    font-weight: 500;
}

.form-buttons button {
    padding: 10px;
    font-size: 16px;
    margin-top: 50px;
}

