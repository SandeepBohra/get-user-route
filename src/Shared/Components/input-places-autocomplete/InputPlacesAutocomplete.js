import React, { Component } from 'react';
import './InputPlacesAutocomplete.css'

class InputPlacesAutocomplete extends Component {
    render() {
        return(
            <div className="InputPlacesAutocomplete">
                <p>{this.props.label}</p>
                <input 
                    type="text" 
                    name={this.props.inputName} 
                    ref={this.props.fieldRef}
                    onChange={this.props.handleInputChange}
                />
            </div>
        )
    }
}

export default InputPlacesAutocomplete;