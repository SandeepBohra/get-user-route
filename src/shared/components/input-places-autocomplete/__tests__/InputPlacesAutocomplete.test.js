import React from 'react';
import renderer from 'react-test-renderer';
import InputPlacesAutocomplete from '../InputPlacesAutocomplete';
import { shallow } from 'enzyme';

describe('<InputPlacesAutocomplete /> component', () => {
  
  it('should match the snapshot', () => {
    const output = renderer.create(<InputPlacesAutocomplete />).toJSON();
    expect(output).toMatchSnapshot();
  })

  it('render HTML correctly', () => {
    const wrapper = shallow(<InputPlacesAutocomplete />);
    
    expect(wrapper.find('input').length).toBe(1);
  });
})