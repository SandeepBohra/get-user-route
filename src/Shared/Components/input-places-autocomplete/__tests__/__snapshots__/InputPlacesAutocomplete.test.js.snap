// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputPlacesAutocomplete /> component should match the snapshot 1`] = `
<div
  className="InputPlacesAutocomplete"
>
  <p />
  <input
    type="text"
  />
</div>
`;
